book/Zapp-Pipes/protocols/login_and_payment_protocol.md

```diff
@@ -0,0 +1,46 @@
+# Login and Payment Protocol
+
+This document outlines the requirements for mapping the data to determine whether login and/or payment are required to gain access to a video item.
```

Comment:
Is it limited to video items? What about articles or other playable items like podcasts?

Reply:
You are right, I will change it. Not limited to videos only.